code/datums/mind.dm

```diff
@@ -88,6 +88,20 @@
 	return language_holder
 
 /datum/mind/proc/transfer_to(mob/new_character, var/force_key_move = 0)
+	if(martial_art)
+		var/mob/living/carbon/human/H = current //Martial arts can only be taught to humans, so it's safe to typecast here without worrying

```

Comment:
I've realized halfway through doing this that mindswap doesnt't play nice with this. Because the mind swaps over and doesn't find the object, it loses the martial art, but the mind swapping to the mob with the martial art's object don't gain that. I need to somehow detect this and assign the martial art to the mob swapping _into_ the one with the object. Any ideas?